src/test/kotlin/racingCar/view/InputViewTest.kt

```diff
@@ -1,35 +1,40 @@
 package racingCar.view
 
 import org.assertj.core.api.Assertions
-import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.Test
 import racingCar.constants.Message
-import java.lang.reflect.InvocationTargetException
-import java.lang.reflect.Method
 
 /**
  * input에 대한 테스트
  * Created by Jaesungchi on 2022.05.07..
  */
 class InputViewTest {
+
+    @Test
+    fun `차량 이름 입력에 null이 입력되었을 때`() {
```

Comment:
테스트 명에는 결과도 함께 넣어주면 어떨까요?

```suggestion
    fun `차량 이름 입력에 null이 입력되면 에러가 발생한다`() {
```

```suggestion
    fun `차량 이름은 필수이다`() {
```